integration-testing/test/conftest.py

```diff
@@ -75,7 +82,7 @@ def make_test_config(request):
 
 
 @contextlib.contextmanager
-def temporary_bonds_file(validator_keys):
+def temporary_bonds_file(validator_keys: List[KeyPair]) -> Iterator[str]:
```

Comment:
The return value is surprising to me.  Isn't there a specialized type for files?

Reply:
Would https://docs.python.org/3/library/typing.html#typing.IO work here?

Reply:
Yeah. It is wrong. It should be `Iterator[IO]`

Reply:
Nonono, the one I wrote is right. IT IS `Iterator[str]`.
The [file](https://github.com/rchain/rchain/pull/1929/files/fa8e119d27d18bd6de33818135af782356edc560#diff-a541af834ea45ee80dba232655f4558eR92) here is a str. Not an IO object.